src/utils/db/db.ts

```diff
@@ -246,9 +246,12 @@ export class DB {
     public async fetchPersons(database: Database = Database.Postgres): Promise<Person[] | ClickHousePerson[]> {
         if (database === Database.ClickHouse) {
             const query = `
-                SELECT * FROM person FINAL JOIN (
+                SELECT * FROM person
+                FINAL
```

Comment:
Do we need final here especially with the join on group by latest query that follows?

Reply:
Good question, not sure, was there before.

Note this function is only used in tests with clickhouse.

Reply:
This is indeed only used in tests... and tests were flakey without it.

Reply:
cool, shouldn't be a problem